.github/CONTRIBUTING.md

```diff
@@ -201,6 +201,18 @@ CREATE DATABASE bookshelf_test;
 
 ### Running the Tests
 
+The recommended way to run the tests is:
+
+- Install [docker & docker-compose](#using-docker-containers)
+- Run the test databases, `docker-compose up -d`
+- Run the tests, `npm test`
+  - You may run the tests many times against the same DB instances.  The tests reset DB state on each run.
+- Teardown the databases, `docker-compose down --remove-orphans`
```

Comment:
This is good advice, but the installation part is already explained above. Maybe that part above could be improved, but duplicating it here doesn't seem right.